kubernetes/cmsweb/scripts/deploy-secrets.sh

```diff
@@ -92,8 +110,9 @@ secretref=${4:-https://openstack.cern.ch:9311/v1/secrets/35145b52-18af-47a3-90d0
         if [ -d $secretdir ] && [ -n "`ls $secretdir`" ]; then
         	for fname in $secretdir/*; do
            	  if [[ $fname == *.encrypted ]]; then
-	       	    sops -d $fname > $secretdir/$(basename $fname .encrypted)
+	       	    ./sops -d $fname > $secretdir/$(basename $fname .encrypted)
 	            fname=$secretdir/$(basename $fname .encrypted)
+		    echo $fname
```

Comment:
If this is not for debugging maybe we can add a message like "Decrypted file $fname"

Reply:
+1